CHANGE_LOG

```diff
@@ -1,9 +1,255 @@
-Version 0.49.0dev
+Version 0.50.0dev
```

Comment:
I think we should defer mentioning 0.50.0dev. We should do this when we tag 0.50.0dev on the master branch.

Reply:
also, so that the release branch will have 0.49.0 as the first entry in the change log

Reply:
fair point, fixed!